packages/react-theme-provider/src/ie11.ts

```diff
@@ -0,0 +1,8 @@
+import { GlobalSettings, isIE11 } from '@fluentui/utilities';
+import { UseVariantClassesOverride, UseVariantClassesOverrideKey } from './makeVariantClasses';
+import { useVariantClassesIE11Override } from './useVariantClassesIE11Override';
+
+// Initialize the global settings override.
+if (isIE11()) {
+  GlobalSettings.setValue<UseVariantClassesOverride>(UseVariantClassesOverrideKey, useVariantClassesIE11Override);
+}
```

Comment:
I don't think we need to drop this on global

Reply:
oh this should be removed.